hack/sigstore-setup/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is copied from sigstore/sigstore
+# See https://github.com/sigstore/sigstore/blob/main/test/e2e/docker-compose.yml
+
+version: "3.8"
+services:
+  vault:
+    image: quay.io/sallyom/vault:latest
```

Comment:
Let's avoid using the `latest` tag.  The image may be updated in the future and we need to make sure that old/stable branches continue to work.

Reply:
can we move these images to a common quay.io directory/repo?  I know we have one for libpod, do we have one for skopeo?

Reply:
Using the `libpod` one should be OK given we need these tests in Podman CI as well.

Reply:
updated from localstack:latest to localstack:0.12.16 (currently used in sigstore)